tests/cpp/scheduler.cpp

```diff
@@ -17,22 +19,17 @@ void clear_finished_sequences(std::vector<SequenceGroup::Ptr>& requests) {
 }
 
 TEST(TestScheduler, general_test) {
-    std::vector<SchedulerConfig> configs{
-        SchedulerConfig {
-        .max_num_batched_tokens = 32,
-        .num_kv_blocks = 6,
-        .block_size = 4,
-        .dynamic_split_fuse = false,
-        .max_num_seqs = 5,
-    },
-        SchedulerConfig {
-        .max_num_batched_tokens = 32,
-        .num_kv_blocks = 6,
-        .block_size = 4,
-        .dynamic_split_fuse = true,
-        .max_num_seqs = 5,
-    }
-    };
+    std::array<SchedulerConfig, 2> configs = {SchedulerConfig(), SchedulerConfig()};
+    configs.at(0).max_num_batched_tokens = 32;
+    configs.at(0).num_kv_blocks = 6;
+    configs.at(0).block_size = 4;
+    configs.at(0).dynamic_split_fuse = false;
+    configs.at(0).max_num_seqs = 5;
+    configs.at(1).max_num_batched_tokens = 32;
+    configs.at(1).num_kv_blocks = 6;
+    configs.at(1).block_size = 4;
+    configs.at(1).dynamic_split_fuse = true;
+    configs.at(1).max_num_seqs = 5;
```

Comment:
Why do we need to change that? The previous version was a bit more readable in my opinion.

Reply:
The current version supports C++17

Reply:
Do you mean C++ 17 doesn't support this initialization?
I was checking with previous version and switched to cpp 17 and didn't see an error while compiling this part.

Reply:
You must use windows to reproduce. gcc is more forgiving